polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -115,6 +121,9 @@ const TICK_TOO_FAR_IN_FUTURE: Tick = 20; // 10 seconds.
 const APPROVAL_DELAY: Tick = 2;
 pub(crate) const LOG_TARGET: &str = "parachain::approval-voting";
 
+// The max number of ticks we delay sending the approval after we are ready to issue the approval
+const MAX_APPROVAL_COALESCE_WAIT_TICKS: Tick = 12;
```

Comment:
To avoid no shows,  this constant has to be kept in line with the `no_show_slots` which is currently on Kusama/Polkadot 3 relay chain blocks - 36 ticks. I would at least add a comment here, or otherwise we should consider the max time to be `1/3 * no_show_slots`

Reply:
See `compute_delayed_approval_sending_tick` here https://github.com/paritytech/polkadot-sdk/blob/alexaggh/feature/approve_multiple_candidates_polkadot_sdk/polkadot/node/core/approval-voting/src/lib.rs#L3596, this the max we can wait but no later than no_show_duration_ticks / 2